pkg/csi/service/wcpguest/controller.go

```diff
@@ -575,13 +575,28 @@ func controllerPublishForBlockVolume(ctx context.Context, req *csi.ControllerPub
 			if err == nil || time.Now().After(timeout) {
 				break
 			}
+			if err != nil {
+				log.Errorf("failed to update virtualmachine. Err: %v", err)
+			}
```

Comment:
Wouldn't we reach line #578 only if there is an error? In that case we can simply log instead of checking for the error?